user/deployment/codedeploy.md

```diff
@@ -89,6 +89,19 @@ deploy:
 
 In this case, S3 deployment provider is not required.
 
+### Waiting for Deployments
+
+By default, the build will continue immediately after triggering a CodeDeploy deploy. To wait for the deploy to complete, use the **wait-until-deployed** option:
+
+{% highlight yaml %}
```

Comment:
Hi @rolebi , would you mind changing these `{highlight}` brackets to triple backtick code markers <pre>```yaml</pre>?

 Thanks! (I know we're not entirely consistently using those yet, but it does make editing easier).